src/chains.jl

```diff
@@ -583,9 +583,14 @@ function header(c::Chains; section=missing)
     )
 
     # Get the timing stats
-    wall = wall_duration(c)
-    compute = compute_duration(c)
-
+    if Base.sym_in(:start_time, keys(c.info)) &&
+        Base.sym_in(:stop_time, keys(c.info))
+        wall = wall_duration(c)
+        compute = compute_duration(c)
+    else
+        wall = missing
+        compute = missing
+    end
```

Comment:
IMO it would be better to just remove the warnings in `to_datetime`. The functions already return the correct values, ie, `missing`, here which clearly indicate that the timings were missing. I don't see why they would ever be useful.

Reply:
Okay. I figured maybe the intent of also printing the type was in case someone passed a `MyCustomButCurrentlyUnsuportedTimeType` object.

Reply:
I've now instead removed the `@warn`.

Reply:
I'm not completely sure what @cpfiffer's (IIRC) original intention was. If we want to keep a warning for unknown types we could alternatively add a method for `::Missing` without warning which would have the same effect in the `show` method here. But since we also don't throw a warning in `to_datetime_vec` but just return `[missing]` it feels fine to remove the warning here as well, in my opinion.